src/doc/rustc/src/platform-support/avr-unknown-unknown.md

```diff
@@ -0,0 +1,75 @@
+# `avr-unknown-unknown`
+
+Series of microcontrollers from Atmel: ATmega8, ATmega328p etc.
+
+**Tier: 3**
+
+## Target maintainers
+
+-[Patryk Wychowaniec](https://github.com/Patryk27) <pwychowaniec@pm.me>
+
+## Requirements
+
+This target is only cross-compiled - x86_64 / aarch64 x Lixux / MacOS hosts are
```

Comment:
`aarch64 x Lixux` doesn't seem quite right :)

Reply:
Hm, why?

Reply:
Shouldn't it be linux?

The `x` just read kind of weird, maybe reword to something like "x86_64 and aarch64 hosts running MacOS or Linux are confirmed to work..."

Reply:
Sure, I've adjusted the sentence to make it more readable.